BUILDING_ON_WINDOWS.md

```diff
@@ -101,14 +101,14 @@ Build results will be placed in a folder at the same level as the "chatterino2"
 
 If you build chatterino, the result directories will contain a `chatterino.exe` file in the `$OUTPUTDIR\release\` directory. This `.exe` file will not directly run on any given target system, because it will be lacking various Qt runtimes.
 
-To produce a standalone package, you need to generate all required files using the tool `windeployqt`. This tool can be found in the `bin` directory of your Qt installation, e.g. at `C:\Qt\5.15.2\msvc2019_64\bin\windeployqt.exe`.
+To produce a standalone package, you will need to generate all required files by using the tool `windeployqt`. This tool can be found in the `bin` directory of your Qt installation, e.g. at `C:\Qt\5.15.2\msvc2019_64\bin\windeployqt.exe`.
 
 To produce all supplement files for a standalone build, follow these steps (adjust paths as required):
 
-1.  Navigate to your build output directory with windows explorer, e.g. `C:\Users\example\src\build-chatterino-Desktop_Qt_5_15_2_MSVC2019_64bit-Release`
+1.  Navigate to your build output directory with Windows Explorer, e.g. `C:\Users\example\src\build-chatterino-Desktop_Qt_5_15_2_MSVC2019_64bit-Release`
 2.  Enter the `release` directory
 3.  Delete all files except the `chatterino.exe` file. You should be left with a directory only containing `chatterino.exe`.
-4.  Open a `cmd` window and execute:
+4.  Open a command prompt (cmd, or PowerShell), and execute:
```

Comment:
```suggestion
4.  Open a command prompt (cmd or PowerShell), and execute:
```

Have you tested these steps with PowerShell?

Reply:
PowerShell is completely backwards compatible with cmd, and only adds features over top of it, so there shouldn't be any issues with using it instead of cmd

Reply:
I don't see how that makes us want to remove `cmd` from the list.

Reply:
false, but ok